src/pages/users.vue

```
@@ -0,0 +1,156 @@
+<template>
+    <div>
+        <Search label="Pretraga korisnika" tooltip="Unesite ime i/ili prezime korisnika" placeholder="Ime Prezime"
+            :on-click="handleSearch" />
+        <v-data-table-virtual :headers="headers" :items="users" height="400" item-value="name" fixed-header>
+            <template v-slot:no-data>
+                Nema pronađenih korisnika.
+            </template>
+            <template v-slot:item.actions="{ item }">
```

ESLint

[error] 'v-slot' directive doesn't support any modifier.

(vue/valid-v-slot)